base_waste_mgmt_data/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    nima_code = fields.Char(string="NIMA Code")
+    auth_number = fields.Char(string="Authorization Number")
```

Comment:
Inspecting at SAU db, these codes seems to be unique for each partners, so we could add a copy=False, in order to prevent errors, @ChristianSantamaria what do you think?

Reply:
Yes, it could be good